rcl_yaml_param_parser/test/test_parse_yaml.cpp

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 #include <stdio.h>
+#include <math.h>
```

Comment:
While you are here, please remove `stdio.h` (it is no longer used once you rebase), and also change this to `#include <cmath>` (since this is a C++ file).

Reply:
Ok